github/Organization.py

```diff
@@ -675,6 +675,46 @@ def create_secret(
         )
         return status == 201
 
+    def add_repository_to_organization_secret(
+        self,
+        secret_name,
+        repository,
+    ):
```

Comment:
These look so close that could combine them? Add another argument for kind/type and assert it's one of actions or dependabot, since everything else is identical.

Further, I also like to support passing in the repository id as an int as well, but that's a nice to have.

Reply:
@s-t-e-v-e-n-k  fixed  can you recheck, please